sdcm/cluster.py

```diff
@@ -1994,11 +1994,8 @@ def offline_install_scylla(self, unified_package, nonroot):
         else:
             additional_pkgs = 'xfsprogs mdadm'
 
-        # scylla current cqlsh still need python2 to work
-        if self.distro.is_ubuntu18:
-            additional_pkgs += ' python2.7'
-        else:
-            additional_pkgs += ' python2'
+        # scylla current cqlsh needs python3 to work
+        additional_pkgs += ' python3'
```

Comment:
it seems that we are using a different branchs for older versions of scylla, so i think it's safe to just bump python in master.

Reply:
I think we fallback to using the relocatable python, so no need to install any other python to make cqlsh work

Reply:
if that's the case, can we just remove this step? as to install python2 for cqlsh is not necessary, and probably misleading.

Reply:
Yes, we could try removing it, and running the test without it.

Reply:
okay. i will repurpose this PR to drop python2 installation. thanks!